algorithms/scaling/outlier_rejection.py

```diff
@@ -71,6 +72,29 @@ def reject_outliers(reflection_table, experiment, method="standard", zmax=6.0):
     return reflection_table
 
 
+def determine_Esq_outlier_index_arrays(Ih_table, experiment, emax=4.0):
+    # first calculate normalised intensities and set in the Ih_table.
+    if not emax:
+        return [flex.size_t() for _ in range(Ih_table.n_datasets)]
```

Comment:
If you make calling this function conditional on non-zero `emax` then you can probably remove this extra check here.